tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -74,15 +77,39 @@
 import py4j.GatewayServer;
 
 import static com.linkedin.tony.Constants.*;
-import static com.linkedin.tony.TonyConfigurationKeys.MLFramework;
+import static com.linkedin.tony.TonyConfigurationKeys.*;
+import static com.linkedin.tony.Utils.*;
 
 
 public class TonyApplicationMaster {
   private static final Log LOG = LogFactory.getLog(TonyApplicationMaster.class);
 
+  /**
+   * Metadata of jobs
+   */
   private ApplicationAttemptId appAttemptID = null;
-  private String appIdString;
+  private static String appIdString;
```

Comment:
why makes this static?

Reply:
See https://github.com/linkedin/TonY/pull/79#discussion_r233597548.

Reply:
Presume this is fixed in the next PR.